### PR TITLE
fixes #10541 - add apipie params for status-trend report

### PR DESCRIPTION
**Pull request by thomasmckay**

### CODE REVIEW

**Comment by ehelms** [comment]
ACK

File: app/services/foreman_gutterball/gutterball_service.rb [hunk 1]
```diff
@@ -65,6 +65,9 @@ def format_query(params)
         end
       end
       params[:owner] = Organization.find(params[:organization_id]).label
+      if params[:subscription_sku]
+        params[:sku] = params.delete(:subscription_sku)
+      end
```
Comment (ehelms) [inline app/services/foreman_gutterball/gutterball_service.rb]: You could reduce this to a single line statement.

**Comment by ehelms** [comment]
ACK either way you go on my comment

**Comment by thomasmckay** [comment]
thanks @ehelms

**Pull request merged**